rc/tools/make.kak

```diff
@@ -53,13 +61,13 @@ define-command -hidden make-open-error -params 4 %{
 define-command -hidden make-jump %{
     evaluate-commands %{
         try %{
-            execute-keys gl<a-?> "Entering directory" <ret><a-:>
+            execute-keys gl<a-?> %opt{make_dir_pattern} <ret><a-:>
             # Try to parse the error into capture groups, failing on absolute paths
-            execute-keys s "Entering directory [`']([^']+)'.*\n([^:/][^:]*):(\d+):(?:(\d+):)?([^\n]+)\z" <ret>l
+            execute-keys s %opt{make_open_dir_error_pattern} <ret>l
```

Comment:
not sure if this is worth the breaking change.
I personally override the make-* commands for projects where I need different behavior

Reply:
What would be the breaking change? The string has just moved to an option?...

Reply:
> I personally override the make-* commands for projects where I need different behavior

I've done similar but it's nice to have consistent behaviors for make.kak and just needing to have to change the patterns... 

Again if there's a breaking change I'd love to figure out how to not make it breaking change :D

Reply:
sorry not on this line but `make_error_pattern` has a new behavior

Reply:
I'll fix it up!